rules/prefer-node-remove.js

```diff
@@ -64,11 +77,23 @@ const create = context => {
 			)) {
 				const argumentName = getArgumentName(node.arguments);
 
+				const {
+					parent: grandparent
+				} = (parent || {});
+
+				let fix = fixer => fixer.replaceText(node, `${argumentName}.remove()`);
+
+				if (parent && ignoredParentTypes.includes(parent.type)) {
+					fix = undefined;
+				} else if (grandparent && ignoredGrandparentTypes.includes(grandparent.type)) {
+					fix = undefined;
+				}
```

Comment:
```suggestion
				if (
					(parent && ignoredParentTypes.includes(parent.type)) || 
					(grandparent && ignoredGrandparentTypes.includes(grandparent.type))
				) {
					fix = undefined;
				}
```

Reply:
Thanks! I've applied this, and also to the append rule.